### PR TITLE
Jiaqi zhu/hw1

### PR DESCRIPTION
**Pull request by Jiaq1Zhu**
Pull request for hw1

### CODE REVIEW

File: ShortQuestions/hw1.md
Comment (Wxl19980214) [inline ShortQuestions/hw1.md:1]: Great answer on question 1. I was confused whether create a README.md or not.

Comment (LichuJiang) [reply ShortQuestions/hw1.md:1]: Agree!

Comment (DaBaLxx) [reply ShortQuestions/hw1.md:1]: Good answers and clear commits.

**Comment by amor-tsai** [comment]
good organization in document structure

**Pull request merged**